### ISSUE TITLE
Table engine does not reflected

### ISSUE THREAD

**Issue by IlyaBel** [xzkostyan/clickhouse-sqlalchemy#90]
I'm trying to reflect the existing table using SQL Expression Language:
```python
from sqlalchemy import create_engine, MetaData, Table

engine = create_engine('clickhouse://default:@localhost:8123/MyDatabase')
metadata = MetaData()
MyTable = Table('MyTable', metadata, autoload=True, autoload_with=engine)
```
But the newly created Table() object does not contains any info about table engine (the engine of an actual table is ReplacingMergeTree). I tried to get the engine info with Reflection Inspector:
```python
from sqlalchemy import create_engine, MetaData, Table
from sqlalchemy.engine import reflection

engine = create_engine('clickhouse://default:@localhost:8123/MyDatabase')
insp = reflection.Inspector.from_engine(engine) 
print(insp.get_table_options('MyTable'))
```
It's possible to get the e.g., MySQL table engine in this way (get_table_options() returns {'mysql_engine': 'InnoDB'}), but I can't do this for ClickHouse. Is there any way?
I'm new to SQLAlchemy and clickhouse-sqlalchemy, so please forgive if I misunderstood something)

**Comment by xzkostyan** [comment]
Unfortunately Engine reflection is not supported yet.

**Comment by xzkostyan** [comment]
Engine reflection is available from 0.1.5 version https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/master/CHANGELOG.md#015---2020-12-14
